querybook/server/lib/query_analysis/templating.py

```diff
@@ -169,16 +229,19 @@ def flatten_recursive_variables(raw_variables: Dict[str, str]) -> Dict[str, str]
     return flattened_variables
 
 
-def get_templated_query_variables(variables_provided):
+def get_templated_query_variables(variables_provided, env=None):
```

Comment:
nice, this is easier to follow now, can you rename env to jinja_env to make it more specific?